doc/project/checklist.md

```diff
@@ -25,10 +25,10 @@ Clock(s)/Reset(s) connected to all sub-modules.
 
 Unit `.sv` exists, meet comportability requirements.
 
-### IP_INSTANCED
+### IP_INSTANTIABLE
 
-Unit is instanced and binding in top level RTL. The unit must not break top
-level functionality such as propagating X through TL-UL interface.
+Unit is able to be instanced and binded in top level RTL.
```

Comment:
Passive tense for "bind" should be "bound" I think.

Reply:
sorry for my bad English :sob: